.flake8

```diff
@@ -0,0 +1,39 @@
+[flake8]
+ignore =
+    # indentation is not a multiple of four,
+    E111,E114,
+    # visually indented line with same indent as next logical line,
+    E129,
+    # line too long
+    E501,
```

Comment:
Instead of suppressing long lines, could we instead just set max-line-length to something reasonable?  The GitHub editor is 127 characters so that allows folks to browse the code online without right-left scrolling.  If we have lines longer that that, we should force them to be wrapped.

Reply:
So I *was* going to do that. The Google Python style guide says [80](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#32-line-length) characters but the code has some lines that are over by a few. So I just disabled checking the line.

TBH, I tried to actually see if I could fix up some of the things in the cpplint file but *really* didn't want to change anything in the cpplint file until we decide what we want to do with it.

I pretty much always use 120 characters.

Reply:
I still makes sense to set the __max-line-length__ to something rather than turning off _all_ line length testing.  I have gotten quite used the [__black__ standard](https://github.com/ambv/black#line-length) which is 88 chars.